src/components/Registration/RegistrationStepThree/RegistrationStepThree.tsx

```diff
@@ -37,23 +41,35 @@ interface IDocumentInfo {
   documentName: string;
 }
 
-interface IDocumentDownloadSectionProps extends IDocumentInfo {
+interface IDocumentDownloadSectionProps
+  extends RouteComponentProps,
+    IDocumentInfo {
   ipaCode: string;
   cookie: string;
-  urlDomainPort: string;
+  showGenericErrorAlert: () => void;
 }
 
 const DownloadDocsSection = (props: IDocumentDownloadSectionProps) => {
+  /**
+   * Create window with custom element _env_ for environment variables
+   */
+  const customWindow = (window as unknown) as ICustomWindow;
+
+  const urlDomainPort =
```

Comment:
`urlDomainPort ` should be passed as a prop. 
generally speaking, components should not extract configuration from
`customWindow._env_`: in case of refactor of the configuration system you'd have to change all components

Reply:
urlDomaiPort is defined here only to fix the problem with getDocuments service, that will be removed when binary files are avaliable (tracked in story https://www.pivotaltracker.com/story/show/169818047). When this problem is fixed, all reference to url domain and port will be included in api-utils -> baseUrlBackendClient